js/modules/loki_snode_api.js

```diff
@@ -31,6 +31,30 @@ const resolveCname = url =>
     });
   });
 
+const setDnsServer = () => {
```

Comment:
https://stackoverflow.com/a/8684009

Reply:
Yeah whoops googled for 'javascript' instead of 'node' :1st_place_medal: